### DISCUSSION TITLE
Quarto website using github pages - new pages don't show up until you go to home page

### DISCUSSION THREAD

**Discussion by anlane611** [quarto-dev/quarto-cli#10824] [Q&A]
### Description

On my [course website](https://anlane611.github.io/ids702-fall24) created with Quarto, if you are on one of the pages other than the home page (e.g., Homework 1), and I then add another page (e.g., Homework 2), the new page doesn't show up simply by refreshing the page. I have to click on the home page (here, IDS 702), for the new pages to show up. Some of my students are finding this to be inconvenient, so I'm wondering if there's a straightforward fix.

**Comment by mcanouil** [comment]
Unfortunately, there is nothing that can be done to help you without the code.

**Comment by cscheid** [comment]
I found the repo to [your website](https://github.com/anlane611/ids702-fall24/).

I called `quarto preview` locally, and I can't reproduce the issue that's on the live website. How are you publishing to github pages? If you're using `quarto publish` then that's a bug we need to investigate, but is it possible that you added homework 2 after publishing the first version of the website and somehow you've only updated HW2.html? HW1.html also needs updating because of the new page on the sidebar.

**Reply by anlane611** [reply]
I used the [Render to Docs](https://quarto.org/docs/publishing/github-pages.html) publishing option, and have now just been committing and pushing to main to update my site.

**Reply by cscheid** [reply]
Have you made sure that you've rendered the entire project? It really looks like somehow `docs/HW1.qmd` hasn't been rerendered since you added `docs/HW2.qmd` to the project. Have you tried deleting the entire `docs` folder and rerunning everything?

**Reply by anlane611** [reply]
Looks like the freeze: auto is the issue. I guess that option doesn't work like I thought it does. Thanks!

**Reply by cscheid** [reply]
@anlane611 This is a bit surprising, because I didn't touch the freeze settings when I attempted to repro on my side. Can you expand a bit on why you think this is the issue?

**Reply by anlane611** [reply]
Sure! When I looked in the _freeze folder, it said that the last version of HW1 was two weeks ago. So to your point, HW1.qmd had not been rendered since adding HW2.qmd. I removed the _freeze folder and removed freeze: auto from the 6-ish most recent documents, and then both Homework 1 and Homework 2 showed up on the side bar without having to go to the home page.
